zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/internal/SessionBuilder.java

```diff
@@ -77,7 +79,12 @@ public static CqlSession buildSession(
     config = config.withBoolean(REQUEST_DEFAULT_IDEMPOTENCE, true);
 
     if (useSsl) config = config.withClass(SSL_ENGINE_FACTORY_CLASS, DefaultSslEngineFactory.class);
-
+    
+    if (overrideHostnameVerification) 
```

Comment:
I think we can write this in one line, excluding the need of if-else.

Replacing this

if (overrideHostnameVerification) 
    	config = config.withBoolean(SSL_HOSTNAME_VALIDATION, true);
    else
    	config = config.withBoolean(SSL_HOSTNAME_VALIDATION, false);

With this

config = config.withBoolean(SSL_HOSTNAME_VALIDATION, overrideHostnameVerification);